typescript/cli/test-configs/zksync/chains/zksync1/metadata.yaml

```
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=../schema.json
+chainId: 260
+displayName: Zksync1
```

consistency with anvil metadata
```suggestion
displayName: zksync1
```